client/src/search/EverythingSearch.js

```diff
@@ -228,6 +228,18 @@ const EverythingSearch = withRouter(
             .map((_child_node, child_key) => this.state[child_key])
             .some()
             .value();
+          const falsified_child_nodes = {
+            ...this.state,
+            ..._.chain(option_node.child_options)
+              .map((_child_node, child_key) => [child_key, false])
+              .fromPairs()
+              .value(),
+          };
+          const is_only_active_node = _.chain(falsified_child_nodes)
+            .filter()
+            .size()
+            .isEqual(0)
+            .value();
```

Comment:
Had to look this over a few times to get it. It's kind of indirect, not clear what's being checked and why e.g. `falsified_child_nodes` is actually `state_with_current_node_chidlren_inactive` or something, but even with a clearer name it's not clear why we need the variable (which is so that all current node children excluded from the following filter?).

I believe something like this is equivalent
```
const is_only_parent_with_active_children = _.every(
  this.state,
  (is_active, option_key) =>  _.has(option_node.child_options, option_key) || !is_active
);
```